### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by vidurb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+    - [Inventory](https://github.com/doublesecretagency/craft-inventory)
+    - [Environment Label](https://github.com/TopShelfCraft/Environment-Label)
+    - [Typogrify](https://github.com/nystudio107/craft-typogrify)
+    - [Relabel](https://github.com/Anubarak/craft-relabel)
+    - [Element Index Defaults](https://github.com/verbb/element-index-defaults)
+    - [CacheFlag](https://github.com/mmikkel/CacheFlag-Craft3)
+* Miranj's preferred JS utilities:
+    - [PJAX](https://github.com/MoOx/pjax)
+    - [lazySizes](https://github.com/aFarkas/lazysizes)
+    - [Web Font Loader](https://github.com/typekit/webfontloader)
```
Comment (rungta) [inline README.md]: Wrong font loader lib.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
+<img src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png">
```
Comment (rungta) [inline README.md]: Why not use the Markdown syntax for the image?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+    - [Environment Label](https://github.com/TopShelfCraft/Environment-Label)
+    - [Typogrify](https://github.com/nystudio107/craft-typogrify)
+    - [Relabel](https://github.com/Anubarak/craft-relabel)
+    - [Element Index Defaults](https://github.com/verbb/element-index-defaults)
+    - [CacheFlag](https://github.com/mmikkel/CacheFlag-Craft3)
+* Miranj's preferred JS utilities:
+    - [PJAX](https://github.com/MoOx/pjax)
+    - [lazySizes](https://github.com/aFarkas/lazysizes)
+    - [Web Font Loader](https://github.com/typekit/webfontloader)
+* And lastly our template hierarchy and organization.
```
Comment (rungta) [inline README.md]: This should perhaps be linked to [Souvik’s presentation on modular architecture](https://miranj.in/blog/2019/modular-architecture-for-building-content-websites).

**Review by rungta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vidurb) [reply README.md]: Have resolved in commit below.

File: README.md (hunk 2 above)
Comment (vidurb) [reply README.md]: The Craft readme had HTML and you guys usually like how they do things. Corrected in commit below.

File: README.md (hunk 3 above)
Comment (vidurb) [reply README.md]: Link added.

**Review by rungta — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+    - [Element Index Defaults](https://github.com/verbb/element-index-defaults)
+    - [CacheFlag](https://github.com/mmikkel/CacheFlag-Craft3)
+* Miranj's preferred JS utilities:
+    - [PJAX](https://github.com/MoOx/pjax)
+    - [lazySizes](https://github.com/aFarkas/lazysizes)
+    - [fontfaceobserver](https://github.com/bramstein/fontfaceobserver)
+    - [fontloader](https://gist.github.com/vidurb/4e2c2cba3affe643ff18bd4eaf869678)
+* And lastly our [template architecture](https://miranj.in/blog/2019/modular-architecture-for-building-content-websites).
+
+More documentation to come.
```
Comment (rungta) [inline README.md]:
```suggestion
```

**Pull request merged**